package.json

```diff
@@ -31,7 +31,7 @@
     "@angular/platform-browser": "~9.1.2",
     "@angular/platform-browser-dynamic": "~9.1.2",
     "@angular/router": "~9.1.2",
-    "@knora/api": "1.0.0-rc.0",
+    "@knora/api": "1.0.0-rc.1",
```

Comment:
Please do not forget to adapt the lib's package.json to "1.0.0-rc.1"

Reply:
@tobiasschweizer sorry but I do not understand this comment

Reply:
The lib has its own `package.json`: https://github.com/dasch-swiss/knora-ui-ng-lib/blob/wip/DSP-142-getPropList/projects/dsp-ui/package.json

Your code requires a specific version of `@knora/api` and because of this it needs to be adapted in the lib's peer deps. When using the lib in another Angular app than the test app, npm can then check if the app env provides the version the lib needs.

there is a good article about this: https://indepth.dev/create-your-standalone-angular-library-in-10-minutes/